src/Admin/AbstractAdmin.php

```diff
@@ -1624,7 +1598,7 @@ final public function isGranted($name, ?object $object = null): bool
         }
 
         $objectRef = null !== $object ? sprintf('/%s#%s', spl_object_hash($object), $this->id($object) ?? '') : '';
-        $key = md5(json_encode($name).$objectRef);
+        $key = md5(json_encode($name, \JSON_THROW_ON_ERROR).$objectRef);
```

Comment:
Not sure if we should avoid this rule too, or not, wdyt @VincentLanglet

Reply:
I would say it's a good practice to throw an error.